### PR TITLE
Proof of concept

### PR DESCRIPTION
**Pull request by andreashasse**

### CODE REVIEW

**Comment by onno-vos-dev** [comment]
Would strongly recommend going with `ex_doc` from the start rather than trying to get that to work later on as it'll be a pain. But we can follow up on it as well :+1:

File: src/rebar3_check_app_calls.app.src [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{application,
+ rebar3_check_app_calls,
+ [{description, "A rebar plugin that checks calls to dependency applications"},
+  {vsn, git},
+  {applications, [kernel, stdlib]},
+  {env, []},
+  {licenses, ["Apache 2.0"]},
+  {links, [{"github", "git@github.com:anha0825/rebar3_check_app_calls.git"}]}]}.
```
Comment (onno-vos-dev) [inline src/rebar3_check_app_calls.app.src]: Is that the correct link? I think https://github.com/andreashasse/rebar3_check_app_calls would be better here?

**Review by onno-vos-dev — APPROVED** [review]

**Review by onno-vos-dev — APPROVED** [review]

**Comment by andreashasse** [comment]
Have to check if otp applications are added correctly.

**Comment by andreashasse** [comment]
> Would strongly recommend going with `ex_doc` from the start rather than trying to get that to work later on as it'll be a pain. But we can follow up on it as well 👍

The edoc thing came from the template I used. I'm not sure if we need to provide documentation of this repo as it isn't a library that others will call.

**Pull request merged**